LICENSE.md

```diff
@@ -1,3 +1,4 @@
+Copyright (c) 2022, The littlefs project.  
```

Comment:
Maybe `The littlefs project authors`?  Then it's copyright by the authors of the project, rather than the project itself.

Reply:
That is a better word choice, though I may shorten it to just `The littlefs authors`

Reply:
Should be updated

Reply:
Looks good!